openc3-ruby/Dockerfile

```diff
@@ -99,6 +106,34 @@ RUN apk update \
 # Note this is needed to fix pip files disappearing in some build environments
 RUN mv /openc3/pip* /openc3/venv/bin/.
 
+# Compile code for libcsp support
+COPY libsocketcan-0.0.12.tar.gz /opt/libsocketcan-0.0.12.tar.gz
+COPY libcsp-6b6e2dd2870bd1f8ab4516123f1fa255d6c1b632.zip /opt/libcsp-6b6e2dd2870bd1f8ab4516123f1fa255d6c1b632.zip
```

Comment:
This is not maintainable. What are these 2 files? Where did they come from? Can this be added to the package_audit?

Reply:
I added the Github source links for each to the comments.   These are rarely updated repos and can be manually updated occasionally or if CVEs popup.